src/content.html

```diff
@@ -3,6 +3,14 @@
   <head>
     <!--characterset UTF-8 Watch this: https://youtu.be/MijmeoH9LT4 -->
 
+    <!-- Google Tag Manager -->
+  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Comment:
google tag manager did mention to put this on each page but i figured for the purposes of sprint 1, it would be okay to just demonstrate it works

Reply:
How do we validate that it works as expected? -- I am asking since I am not too familiar with Google Tag. Is there UI or some report or something we see that validates this?

Reply:
![image](https://user-images.githubusercontent.com/123113761/227400511-5fa2d9af-6ba8-4bcf-943e-224e618101cf.png)
so before publishing, I'll be previewing the page. it works if we get the little consent button for tracking cookies like this photo:
![image](https://user-images.githubusercontent.com/123113761/227400682-1ef564b2-72f8-4dcc-bf8a-a4e4b21d8554.png)

I'm following this tutorial to implement https://www.youtube.com/watch?v=4lKkRD-xVMk